docker-compose.yml

```diff
@@ -1,33 +1,93 @@
-version: '2'
+version: '2.1'
 
 services:
-  mdb:
-    image: mongo:3.4.3
+  # Build: docker-compose build -d bdb
+  # Run: docker-compose run -d bdb
+  mongodb:
+    image: mongo:3.4.13
     ports:
       - "27017"
     command: mongod
-
-  bdb:
+  bigchaindb:
+    depends_on:
+      - mongodb
+      - tendermint
     build:
       context: .
       dockerfile: Dockerfile-dev
       args:
-        backend: mongodb
+        backend: localmongodb
     volumes:
       - ./bigchaindb:/usr/src/app/bigchaindb
       - ./tests:/usr/src/app/tests
       - ./docs:/usr/src/app/docs
-      - ./k8s:/usr/src/app/k8s
       - ./setup.py:/usr/src/app/setup.py
       - ./setup.cfg:/usr/src/app/setup.cfg
       - ./pytest.ini:/usr/src/app/pytest.ini
       - ./tox.ini:/usr/src/app/tox.ini
     environment:
       BIGCHAINDB_DATABASE_BACKEND: localmongodb
-      BIGCHAINDB_DATABASE_HOST: mdb
+      BIGCHAINDB_DATABASE_HOST: mongodb
       BIGCHAINDB_DATABASE_PORT: 27017
       BIGCHAINDB_SERVER_BIND: 0.0.0.0:9984
       BIGCHAINDB_WSSERVER_HOST: 0.0.0.0
+      BIGCHAINDB_START_TENDERMINT: 0
+      BIGCHAINDB_TENDERMINT_HOST: tendermint
+      BIGCHAINDB_TENDERMINT_PORT: 46657
     ports:
       - "9984"
-    command: bigchaindb start
+      - "9985"
+      - "46658"
+    healthcheck:
+      test: ["CMD", "curl", "http://bigchaindb:9984"]
+      interval: 3s
+      timeout: 5s
+      retries: 3
+    command: bigchaindb -l DEBUG start
+  tendermint:
+    image: tendermint/tendermint:0.13
```

Comment:
We probably need to downgrade this version to `0.12`

Reply:
Probably? Should I? :)

Reply:
Yes, /cc @vrde 
In PR  #2138 I found that the event stream works only with `0.12`, the test case fails with `0.13`

Reply:
So that is dependent on our implementation? @kansi Should I downgrade?

Reply:
Yes, we need to downgrade to `0.12` for the Event stream api to work.

Reply:
I figured out the problem with the docker-compose, there was an issue with `config.toml` it still works with `0.13`

Reply:
The reason tests pass with `0.13` is because we never actually use Tendermint in these tests. Furthermore, the event stream test doesn't really check if the connection was a failure i.e. the `result` field in the response json is never asserted [here](https://github.com/bigchaindb/bigchaindb/blob/tendermint/tests/tendermint/test_event_stream.py#L77). I tried to address all these drawbacks [here](https://github.com/bigchaindb/bigchaindb/pull/2138/files#diff-8fe1d9485a354b4d8ace728c749c68b3R72) and it seems that the tests only pass for `0.12`.

Reply:
:face_with_head_bandage:

Thanks @kansi for tracking back the problem. My suggestion is to move forward merging this PR and then work to support the latest version of Tendermint.

Reply:
Changed to `0.12`